hooks/src/index.js

```diff
@@ -44,9 +43,9 @@ options.unmount = vnode => {
 	if (!c) return;
 
 	const hooks = c.__hooks;
-	if (!hooks) return;
-
-	hooks._list.forEach(hook => hook._cleanup && hook._cleanup());
+	if (hooks) {
+		hooks._list.some(hook => hook._cleanup && hook._cleanup());
```

Comment:
what if this returns a truthy value?

Reply:
You are 100% right, I'll fix this tonight

Reply:
@ForsakenHarmony good catch, missed that in my review :+1: